### PR TITLE
Adding postgis in Rails 6 article

### PR DESCRIPTION
**Pull request by SpiderMasterMind**
This is an article about adding PostGIS support to Rails 6

Any feedback is appreciated!

Run `bundle exec jekyll server --watch --drafts` to allow jekyll to serve files in the `drafts` folder so you can check out the formatting. The content column is definitely too narrow. Not sure if we want to update the styling/theme in general - that'll be for another ticket.

### CODE REVIEW

File: _drafts/adding_postgis.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+author: Ben Stacey
+title: Adding PostGIS support to Rails
+description: Steps required to add PostGIS support to Rails 6 with use cases and pre amble
+tags: product engineering
+image: /assets/images/insights-comps.png
+---
+# PostGIS support in Rails 6
+
+Here at BiggerPockets Engineering, we have been busy building the first iteration of the new BPInsights Rent Estimator for Beta launch. [https://biggerpockets.com/insights/property-searches/new] [1]
```
Comment (SpiderMasterMind) [inline _drafts/adding_postgis.md:10]: NOt sure if we wanna call this Rent Estimator now or in here?

File: _drafts/adding_postgis.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
++    - image: circleci/postgres:10-postgis-ram
+```
+
+### Database cleaner
+
+Our database cleaning strategy was truncating the `spatial_ref_sys` table. This table stores constants and reference values required for various PostGIS calculations. If starting a Rails instance throws an `unable to find table: spatial_ref_sys` error you should alter your database cleaning configuration to ignore this table with code like the following:
+
+```diff
++    DatabaseCleaner.clean_with(:deletion, cache_tables: false)
+-    DatabaseCleaner.clean_with(:deletion, cache_tables: false, except: %w(spatial_ref_sys))
```
Comment (lewispb) [inline _drafts/adding_postgis.md:110]: Is this diff the right way round? 🤔

File: _drafts/adding_postgis.md (hunk 1 above)
Comment (lewispb) [reply _drafts/adding_postgis.md:10]: Rendering this locally I'm not sure the reference style links are rendering as we intend?

Screenshot 2020-08-25 at 14 48 29

File: _drafts/adding_postgis.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  )
+end
+
+def self.distance_conversion_node(miles)
+  Arel::Nodes::Multiplication.new(
+    Arel::Nodes.build_quoted(miles),
+    Arel::Nodes::build_quoted(METRES_IN_MILE)
+  )
+end
+private_class_method :distance_conversion_node
```
Comment (lewispb) [inline _drafts/adding_postgis.md:156]:
```suggestion
```

I think we can omit these for clarity in our example here

File: _drafts/adding_postgis.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+def self.st_point_node(long, lat)
+  Arel::Nodes::NamedFunction.new(
+    "ST_POINT",
+    [Arel::Nodes.build_quoted(long), Arel::Nodes.build_quoted(lat)]
+  )
+end
+private_class_method :st_point_node
```
Comment (lewispb) [inline _drafts/adding_postgis.md:164]:
```suggestion
```

I think we can omit these for clarity in our example here

**Review by lewispb — CHANGES_REQUESTED** [review]
Nice! 

Just a couple minor things.

File: _drafts/adding_postgis.md (hunk 2 above)
Comment (SpiderMasterMind) [reply _drafts/adding_postgis.md]: Nice catch 🐟

File: _drafts/adding_postgis.md (hunk 3 above)
Comment (SpiderMasterMind) [reply _drafts/adding_postgis.md]: 👍

File: _drafts/adding_postgis.md (hunk 4 above)
Comment (SpiderMasterMind) [reply _drafts/adding_postgis.md]: 👍

**Comment by SpiderMasterMind** [comment]
Fixed the links as well @lewispb

**Pull request closed without merging**